common/types/socket/types.ts

```diff
@@ -1,5 +1,15 @@
 import { Server } from "socket.io";
 
+export interface EmitWorkTimerArgs {
+	userName: string;
+	roomName: string;
+}
+
+export interface EmitBreakTimerArgs {
+	userName: string;
+	roomName: string;
+}
+
```

Comment:
```suggestion
export interface EmitWorkBreakTimerArgs {
	userName: string;
	roomName: string;
}
```

Since they both have the same args, they can be combined